cloudify_cli/commands/deployments.py

```diff
@@ -312,6 +368,7 @@ def manager_update(ctx,
     if blueprint_id:
         msg += ', using blueprint {0}'.format(blueprint_id)
     logger.info(msg)
+    reinstall_list = list(reinstall_list or [])
```

Comment:
Would this be better as list(reinstall_list) if reinstall_list else []
(to avoid the extra cast to list)

Reply:
no need for the `list()` call at all, actually, it's already a list